src/mb_enhanced_cover_art_uploads/providers/spotify.ts

```diff
@@ -11,4 +12,14 @@ export class SpotifyProvider extends HeadMetaPropertyProvider {
     protected override is404Page(document_: Document): boolean {
         return qsMaybe('head > meta[property="og:title"]', document_) === null;
     }
+
+    protected override fetchPage(url: URL, options?: RequestOptions): Promise<string> {
+        return super.fetchPage(url, {
```

Comment:
Could you add a code comment linking to the forum post here? Otherwise I'll probably forget why this override was needed.